bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/EGateHandler.java

```diff
@@ -223,22 +234,27 @@ private synchronized void sendCommand(String command, int timeout) {
 
     private void pollingConfig() {
         if (!isConnected()) {
+            logger.debug("PollingConfig Run, is not connected so let's connect");
             Socket localEGateSocket = egateSocket;
             BufferedWriter localWriter = writer;
             if (localEGateSocket == null || localWriter == null) {
+                logger.debug("Error eGateSocket null, writer null in pollingConfig(), returning...");
                 return;
             }
+
             synchronized (lock) {
                 try {
-                    localEGateSocket.connect(new InetSocketAddress(host, port));
-                    localEGateSocket.setSoTimeout(SOCKET_TIMEOUT_SEC);
+                    localEGateSocket.connect(new InetSocketAddress(host, port), SOCKET_TIMEOUT_SEC);
+                    updateStatus(ThingStatus.ONLINE);
+                    logger.debug("pollingConfig() successsully connected {}", localEGateSocket.isClosed());
                     localWriter.write("SilenceModeSet;Value=0;" + CR);
                     localWriter.flush();
                 } catch (IOException e) {
                     logger.debug("IOException in pollingConfig: {} host {} port {}", e.toString(), host, port);
                     try {
                         localEGateSocket.close();
                         egateSocket = null;
+                        logger.debug("EGate closed");
                         updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.toString());
```

Comment:
No need to set the thing status here, it will be set again just below at line 270.

Reply:
Fixed, thank for that hint